bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup/KeyCeremonyMediator.cs

```diff
@@ -739,7 +795,7 @@ private async Task RunStep5()
         {
             guardian!.SaveElectionPartialKeyBackup(backup.Backup!);
             var verification = guardian.VerifyElectionPartialKeyBackup(backup.GuardianId!, keyCeremonyId);
-            if (verification == null)
+            if (verification == null || !verification.Verified)

```

Comment:
This change is good, but will require UI changes to catch. Right now it interprets the value. If not reverted, this will create a regression.

Reply:
Future: UI needs to handle a throw on failed verification, and we need to throw on failed verification.

Reply:
we can switch it back for now if oyu want just say the word

Reply:
def. I'll add an issue, because this is a good, but breaking change.

Reply:
#213